examples/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-base-amp/styles/base/base.less

```diff
@@ -57,3 +57,19 @@ a {
         outline: none;
     }
 }
+
+/*
+ * Making sure all images included via the image component have the proper css fallback
+ */
+
+.cmp-image {
```

Comment:
I worry that this may become too large and hard of a file to navigate if we place _all_ amp component styles in `base.less`. I wonder if we should break amp styles out into their own file. I would even be happy with just a `amp-components.less` (or something similar) to place them all in. To me, I don't necessarily know unless I have knowledge of this project, that the components-examples directory houses all of the amp styles.

Reply:
@paigekeenan , great comment though. I think this will be unique to components that are reused by other components. The image component is used by others like Teaser. which can be a problem right now. i.e if the teaser component gets placed in a page and the image component standalone is not, the amp markup will be there but not the CSS as it is local to the image component. by adding some of this to the base clientlib, we add better support for these situations.
Based on your comment though, for a more self-documenting structure, I went ahead and separated out the image specific css in its own LESS file separated in a `components`  folder in the base clientlibs. This also follows the AEM core component practice of their clientlibs.
See my last commit. Thanks!